modules/organization-policy/README.md

```diff
@@ -0,0 +1,166 @@
+            description = "Condition description" # (Optional)
+            expression  = "Condition expression"  # (Optional) For example "resource.matchTag('123456789/environment', 'prod')".
+            location    = "policy-error.log"      # (Optional) String indicating the location of the expression for error reporting.
+            title       = "condition-title"       # (Optional)
+          }
+          values = {             # (Optional) Only for list constraints. List of values to be used for this PolicyRule.
+            allowed_values = ["value1", "value2"] # (Optional) List of values allowed at this resource.
+            denied_values  = ["value3", "value4"] # (Optional) List of values denied at this resource.
+          }
+        }
+      ]
+    }
+  }
+}
+# tftest skip
+```
+
+### Structure of configuration provided in a yaml file/s
+
+Configuration should be placed in a set of yaml files in the config directory. Policy entry structure as follows:
+
+```yaml
+parent_id: # parent id in format projects/project-id, folders/1234567890 or organizations/1234567890.
+  policy_name1: # policy constraint id, for example compute.vmExternalIpAccess.
+    inherit_from_parent: true|false # (Optional) Only for list constraints. Determines the inheritance behavior for this policy.
+    reset: true|false               # (Optional) Ignores policies set above this resource and restores the constraint_default enforcement behavior.
+    rules:
+      - allow_all: true|false # (Optional) Only for list constraints. Setting this to true means that all values are allowed.
+        deny_all: true|false  # (Optional) Only for list constraints. Setting this to true means that all values are denied.
+        enforce: true|false   # (Optional) Only for boolean constraints. If true, then the Policy is enforced.
+        condition:            # (Optional) A condition which determines whether this rule is used in the evaluation of the policy.
+          description: Condition description   # (Optional)
+          expression: Condition expression     # (Optional) For example resource.matchTag("123456789/environment", "prod")
+          location: policy-error.log           # (Optional) String indicating the location of the expression for error reporting.
+          title: condition-title               # (Optional)
+        values:               # (Optional) Only for list constraints. List of values to be used for this PolicyRule.
+          allowed_values: ['value1', 'value2'] # (Optional) List of values allowed at this resource.
+          denied_values:  ['value3', 'value4'] # (Optional) List of values denied at this resource.
+
+```
+
+Module allows policies to be distributed into multiple yaml files for a better management and navigation.
+
+```bash
+├── org-policies
+│   ├── baseline.yaml
+│   ├── image-import-projects.yaml
+│   └── exclusions.yaml
+```
+
+Organization policies example yaml configuration
+
+```bash
+cat ./policies/baseline.yaml
```

Comment:
what's in the other files?

Reply:
The same, you may just differentiate into multiple files like
org-level.yaml
folder-level.yaml
or env based, it's simply for a better navigation and helps keep PRs more readable.